consensus/consensus_test.go

```diff
@@ -650,6 +699,28 @@ func TestValidateBlockBody(t *testing.T) {
 				assert.Equal(t, err, expected)
 			},
 		},
+		{
+			"EmptyTx", func(t *testing.T) {
```

Comment:
Maybe this can be renamed to `ZeroGasTx` since it's focusing on sending a tx with 0 gas